sdk/src/AbstractMain.cpp

```diff
@@ -225,19 +212,25 @@ int AbstractMain::run(int in_argc, char** in_argv)
    if (ret != 0)
       return ret;
 
-   // Remove the stderr log destination.
-   removeLogDestination(stderrLogDest->getId());
-
-   if (options.getLogLevel() > LogLevel::INFO)
-   {
-      addLogDestination(
+   addLogDestination(
```

Comment:
Let's move this `addLogDestination` call here to just after `setProgramId` above on line 198. Effectively, we're replacing the syslog destination with a file one, as our products log to file by default now.